nfs/Dockerfile

```diff
@@ -0,0 +1,52 @@
+FROM --platform=$BUILDPLATFORM golang:1 AS builder
+
+WORKDIR /src
+COPY --link go.mod go.sum /src/
+RUN go mod download
+COPY --link ./cmd /src/cmd
+COPY --link ./pkg /src/pkg
+
+ARG TARGETARCH
+ARG TARGETOS
+ARG VERSION=unknown
+RUN --mount=type=cache,target=/root/.cache/go-build GOOS=$TARGETOS GOARCH=$TARGETARCH CGO_ENABLED=0 \
+    go build \
+    -a \
+    -ldflags "-X github.com/piraeusdatastore/linstor-csi/pkg/driver.Version=$VERSION -extldflags -static"  \
+    -o /nfs-helper  \
+    ./cmd/nfs-helper
+
+FROM debian:bookworm
+
+RUN { echo 'APT::Install-Recommends "false";' ; echo 'APT::Install-Suggests "false";' ; } > /etc/apt/apt.conf.d/99_piraeus
+
+RUN --mount=type=cache,target=/var/cache,sharing=private \
+    --mount=type=cache,target=/var/lib/apt/lists,sharing=private \
+    --mount=type=tmpfs,target=/var/log \
+    apt-get update && \
+    apt-get install -y wget ca-certificates && \
+    wget https://packages.linbit.com/public/linbit-keyring.deb -O /var/cache/linbit-keyring.deb && \
```

Comment:
does not matter too much, especially not for debian, but whenever possible this should use a distro specific keyring from https://packages.linbit.com/public/linbit-keyring/

Reply:
Thanks! Did not know about https://packages.linbit.com/public/linbit-keyring/ :sweat_smile: